facechain/train_text_to_image_lora.py

```diff
@@ -46,7 +46,8 @@
 from datasets import load_dataset
 from diffusers import (AutoencoderKL, DDPMScheduler, DiffusionPipeline,
                        DPMSolverMultistepScheduler,
-                       StableDiffusionInpaintPipeline, UNet2DConditionModel)
+                       StableDiffusionInpaintPipeline, UNet2DConditionModel, StableDiffusionXLPipeline)
```

Comment:
create a new file train_text_to_image_sdxl_lora.py for sdxl finetune

Reply:
done